Doesn't build with maven command line

`mvn -DskipTests verify` gives error:

```
[INFO] --- minify:1.7.6:minify (minify) @ mapClient ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.843 s
[INFO] Finished at: 2024-03-04T10:30:48Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.samaxes.maven:minify-maven-plugin:1.7.6:minify (minify) on project mapClient: Execution minify of goal com.samaxes.maven:minify-maven-plugin:1.7.6:minify failed: A required class was missing while executing com.samaxes.maven:minify-maven-plugin:1.7.6:minify: org/codehaus/plexus/util/DirectoryScanner
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>com.samaxes.maven:minify-maven-plugin:1.7.6
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/mase/.m2/ext_repository/com/samaxes/maven/minify-maven-plugin/1.7.6/minify-maven-plugin-1.7.6.jar
[ERROR] urls[1] = file:/home/mase/.m2/ext_repository/com/yahoo/platform/yui/yuicompressor/2.4.8/yuicompressor-2.4.8.jar
[ERROR] urls[2] = file:/home/mase/.m2/ext_repository/rhino/js/1.7R2/js-1.7R2.jar
[ERROR] urls[3] = file:/home/mase/.m2/ext_repository/com/google/javascript/closure-compiler/v20161024/closure-compiler-v20161024.jar
[ERROR] urls[4] = file:/home/mase/.m2/ext_repository/com/google/javascript/closure-compiler-externs/v20161024/closure-compiler-externs-v20161024.jar
[ERROR] urls[5] = file:/home/mase/.m2/ext_repository/args4j/args4j/2.33/args4j-2.33.jar
[ERROR] urls[6] = file:/home/mase/.m2/ext_repository/com/google/guava/guava/19.0/guava-19.0.jar
[ERROR] urls[7] = file:/home/mase/.m2/ext_repository/com/google/protobuf/protobuf-java/3.0.2/protobuf-java-3.0.2.jar
[ERROR] urls[8] = file:/home/mase/.m2/ext_repository/com/google/code/findbugs/jsr305/3.0.1/jsr305-3.0.1.jar
[ERROR] urls[9] = file:/home/mase/.m2/ext_repository/com/google/jsinterop/jsinterop-annotations/1.0.0/jsinterop-annotations-1.0.0.jar
[ERROR] urls[10] = file:/home/mase/.m2/ext_repository/com/google/code/gson/gson/2.8.0/gson-2.8.0.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
[ERROR]
[ERROR] -----------------------------------------------------
[ERROR] : org.codehaus.plexus.util.DirectoryScanner
```

--- a/pom.xml
+++ b/pom.xml
@@ -216,6 +216,13 @@
                 <groupId>com.samaxes.maven</groupId>
                 <artifactId>minify-maven-plugin</artifactId>
                 <version>1.7.6</version>
+                <dependencies>
+                  <dependency>
+                    <groupId>org.codehaus.plexus</groupId>
+                    <artifactId>plexus-utils</artifactId>
+                    <version>3.5.1</version>
+                  </dependency>
+                </dependencies>
                 <executions>
                     <execution>
                         <id>minify</id>

